docs/how_to_guide.rst

```diff
@@ -0,0 +1,138 @@
+RBAC authorization guide
+========================
+
+This section covers information you need to know to implement RBAC authorization on API endpoints.
+
+
+.. contents:: Table of Contents
+
+
+Roles
+-----
+* system wide roles:
+    These are the roles used across all the edx services. Role data is added by LMS using a mapping
+    between a user and a role. Role data is communicated by JSON Web Tokens.
+* feature specific roles:
+    These are specific to a feature/service and role assignment is created through django admin by a specific service.
+
+Access
+------
+* Implicit:
+    Verify the request's user access by mapping user's system wide roles found in JWT to local feature roles.
+* Explicit:
+    Verify if there is a role assignment for a given user and role. Explicit access is used when we manually want
+    to give access to a user to a specific resource without creating any role data in JWT.
+
+.. note::
+In both the implicit and explicit access, role data has a context that tells which resource(s) the requesting user can access
+
+Implementation
+--------------
+To add RBAC implicit and explicit authorization checks, you need to follow the below steps. Here we are using ``LMS``,
+`edx-enterprise-data <https://github.com/edx/edx-enterprise-data/>`_ and `ecommece <https://github.com/edx/ecommerce>`_
+codebases as an example.
+
+1. In LMS create a `system wide role data migration <https://github.com/edx/edx-enterprise/blob/master/enterprise/migrations/0066_add_system_wide_enterprise_operator_role.py>`_. You only need to do this if you are creating a new role.
```

Comment:
Can you explain when we would want to introduce a system wide role and when we wouldn't?